src/router/v1/auth.py

```
@@ -89,3 +89,12 @@ async def send_reset_password_confirm_email(
 ):
     verify_token = await _auth_service.send_reset_password_confirm_email(db, email)
     return res_success(msg='email sent', data={'token': verify_token})
+
+
+@router.delete('/accounts', status_code=status.HTTP_200_OK)
+async def delete_account(
+    payload: auth.DeleteAccountDTO = Body(...),
```

Ruff (0.15.9)

[warning] Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)